pkg/apis/operator/v1alpha1/validation/validation.go

```diff
@@ -129,6 +129,11 @@ func validateRuntimeCluster(runtimeCluster operatorv1alpha1.RuntimeCluster, fldP
 		}
 	}
 
+	// TODO(scheererj): Turn this into a native CRD validation as soon as the `ingress.domain` field was dropped (planned after v1.88)
```

Comment:
same here?

Reply:
Changed the comment to 'after v1.90' as suggested with https://github.com/gardener/gardener/pull/9038/commits/e623c777f0a6fa66b208296f382d5700906b4ee6.